site/sfguides/src/getting_started_with_azure_openai_and_snowflake/getting_started_with_azure_openai_and_snowflake.md

```diff
@@ -196,12 +196,13 @@ Duration: 10
 Head back to your AzureML Workspace and click on the notebook blade and select open terminal and start/select your compute instance near the top of the screen. In the terminal run the below code to copy the notebook that will use to orchestrate inference.
 
 ```
-git clone https://github.com/sfquickstarts/tree/master/site/sfguides/src/getting_started_with_azure_openai_and_snowflake/assets/openai-nb
+git clone https://github.com/Snowflake-Labs/sf-samples.git
```

Comment:
My .02 – I think you should just link to the file directly and ask folks to download it, vs. asking that they clone all of the sf-samples to their computer. What do you think? @marzillo-snow

Reply:
For example:

"Download the notebook associated with this Quickstart. You can find the notebook here: https://github.com/Snowflake-Labs/sf-samples/blob/main/samples/openai-nb/openai.ipynb. Download it using the icon in the top right."

Etc.